.taskcluster.yml

```diff
@@ -0,0 +1,52 @@
+version: 0
+
+metadata:
+  name: Normandy
+  description: Normandy CI Tasks
+  owner: "{{ event.head.user.email }}"
+  source: "{{ event.head.repo.url }}"
+
+tasks:
+  - metadata:
+      name: normandy:decision
+      description: Normandy decision task
+      owner: "{{ event.head.user.email }}"
+      source: "{{ event.head.repo.url }}"
+    provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - queue:create-task:aws-provisioner-v1/github-worker
+    payload:
+      env:
```

Comment:
Ah, oops. This is actually because there used to be some subkeys, like this:

```yaml
payload:
  env:
    GECKO_DEV_REPO_URL: https://github.com/mozilla/gecko-dev
```

I removed the environment variables without removing the env key. I'll remove it.